src/LinearMap.jl

```diff
@@ -5,7 +5,7 @@ export LinearMap,
        constraints_list
 
 """
-    LinearMap{N<:Real, S<:LazySet{N}, NM, MAT<:AbstractMatrix{NM}} <: LazySet{N}
+    LinearMap{N<:Real, S<:LazySet{N}, MN, MAT<:AbstractMatrix{MN}} <: LazySet{N}
```

Comment:
It wasn't called `MN`, since `MN` is used for Matrices of numeric type `N` as used above in `ProjectionSparseMatrixExp` and [in the gudelines](https://juliareach.github.io/JuliaReachDevDocs/latest/guidelines/#Coding-Guidelines-1]. To change something, we could change `MAT` to `MN` but then need another letter for the numeric type of the matrix.

Reply:
Okay, I will drop the second commit then.